release-1.8/release-1.8.md

```diff
@@ -0,0 +1,81 @@
+# Proposed timeline for 1.8
+We will begin the 1.8 release the first week of July 2017.
+
+The proposal below is similar in layout to the 1.7 plans with some
+modifications:
+- as before, key days aren't Fridays, since it can be hard to end milestones right up against weekends
+- NEW in 1.8: alpha's should not be cut unless all release tests are passing
+
+## 1.8 Release Schedule
+- July 5 (Weds) coding start (7w)
+- July 12th (Weds): v1.8.0-alpha.1
+- July 26th (Weds): v1.8.0-alpha.2
+- Aug 1 (Tues): Feature Freeze
+- Aug 9 (Weds): v1.8.0-alpha.3
+- Aug 23 (Weds): v1.8.0-alpha.4
```

Comment:
Aug 8: release notes draft for features due
Aug 15: hard deadline for feature related bug fixes
- Consequence: your feature not included in release notes